ibis/omniscidb/client.py

```diff
@@ -649,7 +586,8 @@ def __init__(
 
     def __del__(self):
         """Close the connection when instance is deleted."""
-        self.close()
+        if hasattr(self, 'con') and self.con:
```

Comment:
why woudn't this always be true?

Reply:
I didn't investigate too much the reason. but I had this issue many times playing with the client.